lib/api/controllers/bulkController.js

```diff
@@ -171,6 +172,26 @@ class BulkController extends NativeController {
     return { deleted };
   }
 
+  /**
+   * Directly update every documents matching the search query without:
+   *  - applying max documents write limit
+   *  - injecting Kuzzle metadata
+   *  - triggering realtime notifications
```

Comment:
Weird, sometimes you used 'inject' and others you used 'add'
```suggestion
   *  - Applying max documents write limit
   *  - Injecting Kuzzle metadata
   *  - Triggering realtime notifications
```